src/gp_engine.hpp

```diff
@@ -46,7 +46,7 @@ class GPEngine {
   void SetBranchLengthsToConstant(double branch_length) {
     branch_lengths_.setConstant(branch_length);
   };
-  void SetSBNParameters(EigenVectorXd q) { q_ = std::move(q); };
+  void SetSBNParameters(EigenVectorXd&& q) { q_ = std::move(q); };
```

Comment:
Note @junseonghwan if we want to be able to move using this method we need to pass an rvalue reference.